e2e/playwright/snapshot-tests.spec.ts

```diff
@@ -704,12 +703,12 @@ test.describe(
         },
         {
           settingsKey: TEST_SETTINGS_KEY,
-          settings: TOML.stringify({
+          settings: settingsToToml({
```

Comment:
fwiw I'm a little spooked `TOML.stringify === settingsToToml` but if it works it works I guess

Reply:
it makes it way more type safe

Reply:
thats the reason why i did it so no one puts in bad json

Reply:
Oh nice ok